<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page1</class>
 <widget class="QWidget" name="page1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>551</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DM Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="instruction_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DM Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CHOOSE TYPE OF CIS BENCHMARK YOU WANT TO RUN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="options_hlayout">
     <item>
      <widget class="QGroupBox" name="machine_type_box">
       <property name="font">
        <font>
         <family>DM Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>MACHINE TYPE</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>133</width>
          <height>62</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="machine_type_vlayout">
         <item>
          <widget class="QRadioButton" name="workstation_btn">
           <property name="font">
            <font>
             <family>DM Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>WORKSTATION</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="server_btn">
           <property name="font">
            <font>
             <family>DM Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>SERVER</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item alignment="">
      <widget class="QGroupBox" name="benchmark_level_box">
       <property name="font">
        <font>
         <family>DM Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>BENCHMARK LEVEL</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>95</width>
          <height>62</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="benchmark_level_vlayout">
         <item>
          <widget class="QRadioButton" name="level1_btn">
           <property name="font">
            <font>
             <family>DM Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LEVEL 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="level2_btn">
           <property name="font">
            <font>
             <family>DM Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LEVEL 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottom_btns">
     <item alignment="">
      <widget class="QPushButton" name="sysInfo_btn">
       <property name="font">
        <font>
         <family>DM Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SYSTEM INFORMATION</string>
       </property>
      </widget>
     </item>
     <item alignment="">
      <widget class="QPushButton" name="continue_btn">
       <property name="font">
        <font>
         <family>DM Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CONTINUE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
